docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -242,6 +242,51 @@ Example:
 sharedStorageClass: "bulk|foo.*"
 ----
 
+== `thanos`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+Configures the thanos sidecars object storage.
+If this dict is empty, the thanos sidecar won't be deployed.
+See the https://thanos.io/tip/thanos/storage.md[Official documentation] for all possible configuration options.
+
+Example:
+
+[source,yaml]
+---
+thanos:
+  type: S3
+  config:
+    bucket: my-bucket
+    endpoint: my-s3.example.com
+    access_key: ?{vaultkv:${cluster:tenant}/${cluster:name}/thanos/access_key}
+    secret_key: ?{vaultkv:${cluster:tenant}/${cluster:name}/thanos/secret_key}
+---
+
+== `rules`
+
+[horizontal]
+type:: dict
+default:: `{}`
+
+A list of additional rules to add.
+
+Example:
+
+[source,yaml]
+---
+rules:
+  generic-rules:
+  - alert: ContainerOOMKilled
+    annotations:
+      message: A container ({{$labels.container}}) in pod {{ $labels.namespace }}/{{ $labels.pod }} was OOM killed
+    expr: |
+      kube_pod_container_status_last_terminated_reason{reason="OOMKilled"} == 1
+    labels:
+      source: https://git.vshn.net/swisscompks/syn-tenant-repo/-/blob/master/common.yml
+      severity: devnull
```

Comment:
This would be the format for additional alerting rules when using my suggestion

```suggestion
  generic-rules:
    "alert:ContainerOOMKilled":
      annotations:
        message: A container ({{$labels.container}}) in pod {{ $labels.namespace }}/{{ $labels.pod }} was OOM killed
      expr: |
        kube_pod_container_status_last_terminated_reason{reason="OOMKilled"} == 1
      labels:
        source: https://git.vshn.net/swisscompks/syn-tenant-repo/-/blob/master/common.yml
        severity: devnull
```